Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al

```diff
@@ -250,34 +250,73 @@ codeunit 130456 "Test Suite Mgt."
     procedure GetTestMethods(var ALTestSuite: Record "AL Test Suite"; var AllObjWithCaption: Record AllObjWithCaption)
     var
         TestLineNo: Integer;
+        DialogContent: Label '-- Get Test Methods --\#1#\#2#';
+        i: Integer;
```

Comment:
Also minor - we use capital naming convention (PascalCase), thus it should be I and full names for variables, n -> TotalNumberOfTests or AllObjWithCaptionCount and etc., d - ProgressWindow or similar naming we have in the product.

Reply:
OK, I will do this change in all new DialogContent Labels, including the change from Switching to Inverting as @JesperSchulz  has proposed above.

Reply:
I understand the style rules, but I try to avoid unnecessary verbosity... 

If you all see it correctly, I will rename these variables as follows using more agnostic words:
i -> Counter
n -> TotalCount
d -> Window (already used in standard code too)